suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -86,14 +86,17 @@ export const prepareAccountsReducer = createReducerWithExtraDeps(
                 state.push(account);
             })
             .addCase(accountsActions.createIndexLabeledAccount, (state, action) => {
-                const { deviceState, symbol } = action.payload;
-                const deviceNetworkAccounts = state.filter(
-                    account => account.deviceState === deviceState && account.symbol === symbol,
+                const { deviceState, symbol, accountType } = action.payload;
```

Comment:
We should let QA test this on desktop before merge.

Reply:
I think this is only used in mobile right now.

Reply:
But isn't it redundant? If I check things correctly, this is only called from our discoveryThunks, where it's not passed as an parameter.

Reply:
@Nodonisko Bohdan is right. This action was created specifically for suite-native usecase, so there is no need for desktop QA testing.